services/m3dbnode/main/main.go

```diff
@@ -75,32 +79,29 @@ func main() {
 		}
 	}
 
-	clientOpts, err := server.DefaultClientOptions(id, tchannelNodeAddr, shardSet)
-	if err != nil {
-		log.Fatalf("could not create client options: %v", err)
-	}
-
-	c, err := client.NewClient(clientOpts)
+	cli, err := client.NewClient(server.DefaultClientOptions(topoInit))
```

Comment:
probably should call topoInit.Init() here and reuse the Topology in both client and database options?